tests/unittests/OperatorTest.cpp

```diff
@@ -26,22 +26,44 @@
 #include "llvm/Support/raw_ostream.h"
 
 using namespace glow;
+
 class Operator : public ::testing::TestWithParam<BackendKind> {
 public:
   Operator() : mod_(EE_.getModule()) { F_ = mod_.createFunction("main"); }
 
   ~Operator() override { mod_.clear(); }
 
 protected:
+  bool isEnabledBackend(const std::set<BackendKind>& enabledBackends) {
+    return enabledBackends.find(GetParam()) != enabledBackends.end();
+  }
+
+  const BackendKind Interpreter = BackendKind::Interpreter;
+  const BackendKind CPU = BackendKind::CPU;
+  const BackendKind OpenCL = BackendKind::OpenCL;
+
   ExecutionEngine EE_{GetParam()};
   Module &mod_;
   Function *F_;
   Context ctx_;
 };
 
-class InterpAndCPU : public Operator {};
+// The parameterized test suite is instantiated with all available backends.
+INSTANTIATE_TEST_CASE_P(AllOperatorTest, Operator,
+                        ::testing::Values(
+#ifdef GLOW_WITH_CPU
```

Comment:
If we move to using this `ENABLED_BACKENDS()` pattern in other similarly parameterized tests (e.g. `QuantizationTest`, `BackendCorrectnessTest`, etc.), then perhaps we should macro this declaration of all backends and place it in `BackendTestUtils.h`, so we don't need to ifdef etc. for every backend in every test that uses this. Also then maybe we'd move `isEnabledBackend()` and `ENABLED_BACKENDS()` there too.

Reply:
Sounds like a good plan. Can we check in this change first and apply it in a follow-up PR? An incremental change would be easier to make.